solidity/deploy/07_deploy_reimbursement_pool.ts

```diff
@@ -0,0 +1,38 @@
+import { HardhatRuntimeEnvironment } from "hardhat/types"
+import { DeployFunction } from "hardhat-deploy/types"
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  const { deployments, getNamedAccounts, ethers } = hre
```

Comment:
Is this approach kosher or not @nkuba?

Reply:
As long as the `gas-reporter` works maybe we can leave it as is? There's nothing wrong with this implementation, only if it breaks the `gas-reporter`, which it doesn't (at least for now).

Reply:
See https://github.com/keep-network/keep-core/issues/2979